requirements.txt

```diff
@@ -0,0 +1,121 @@
+alabaster==0.7.12
+aperturedb==0.1.1
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+asgiref==3.5.0
+astroid==2.11.0
+asttokens==2.0.5
+attrs==21.4.0
+Babel==2.9.1
+backcall==0.2.0
+backports.zoneinfo==0.2.1
+black==22.1.0
+bleach==4.1.0
+boto3==1.21.7
+botocore==1.24.7
+certifi==2021.10.8
+cffi==1.15.0
+charset-normalizer==2.0.12
+click==8.0.4
+cycler==0.11.0
+debugpy==1.5.1
+decorator==5.1.1
+defusedxml==0.7.1
+Django==4.0.2
+docutils==0.17.1
+entrypoints==0.4
+executing==0.8.2
+fonttools==4.29.1
+idna==3.3
+image==1.5.33
+imageio==2.16.0
+imagesize==1.3.0
+importlib-metadata==4.11.2
+importlib-resources==5.4.0
+iniconfig==1.1.1
+ipykernel==6.9.1
+ipython==8.0.1
+ipython-genutils==0.2.0
+jedi==0.18.1
+Jinja2==3.0.3
+jmespath==0.10.0
+jsonschema==4.4.0
+jupyter-client==7.1.2
+jupyter-core==4.9.2
+jupyterlab-pygments==0.1.2
+kiwisolver==1.3.2
+lazy-object-proxy==1.7.1
+MarkupSafe==2.1.0
+matplotlib==3.5.1
+matplotlib-inline==0.1.3
+mistune==0.8.4
+mypy-extensions==0.4.3
+nbclient==0.5.11
+nbconvert==6.4.2
+nbformat==5.1.3
+nest-asyncio==1.5.4
+networkx==2.7rc1
+notebook==6.4.8
+numpy==1.22.2
+opencv-python==4.5.5.62
+packaging==21.3
+pandas==1.4.1
+pandocfilters==1.5.0
+parso==0.8.3
+pathspec==0.9.0
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==9.0.1
+platformdirs==2.5.1
+pluggy==1.0.0
+prometheus-client==0.13.1
+prompt-toolkit==3.0.28
+protobuf==4.0.0rc2
+ptyprocess==0.7.0
+pure-eval==0.2.2
+py==1.11.0
+pycparser==2.21
+pydantic==1.9.0
+Pygments==2.11.2
+pyparsing==3.0.7
+pyrsistent==0.18.1
+pytest==7.0.1
+python-dateutil==2.8.2
+pytz==2021.3
+PyWavelets==1.2.0
+PyYAML==6.0
+pyzmq==22.3.0
+requests==2.27.1
+s3transfer==0.5.2
+scikit-image==0.19.2
+scipy==1.8.0
+Send2Trash==1.8.0
+six==1.16.0
+snowballstemmer==2.2.0
+Sphinx==4.4.0
+sphinx-autoapi==1.8.4
+sphinx-rtd-theme==1.0.0
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==2.0.0
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.5
+sqlparse==0.4.2
+stack-data==0.2.0
+terminado==0.13.1
+testpath==0.6.0
+tifffile==2022.2.9
+tomli==2.0.1
+torch==1.10.2
+torchvision==0.11.3
+tornado==6.1
+traitlets==5.1.1
+typing-extensions==4.1.1
+Unidecode==1.3.4
+urllib3==1.26.8
+vdms==0.0.16
+wcwidth==0.2.5
+webencodings==0.5.1
+wrapt==1.14.0
+zipp==3.7.0
```

Comment:
Just curious: do the versions have to be specific, i.e. `==`, rather than `>=`?

Reply:
I am sure the versions won't even matter. I'll reduce this to what I specifically installed to have this running rather than the output of pip freeze.